src/test/kotlin/uk/gov/dluhc/printapi/rds/mapper/InstantMapperTest.kt

```diff
@@ -66,4 +67,21 @@ class InstantMapperTest {
             assertThat(actual).isNull()
         }
     }
+
+    @Nested
+    inner class FromInstantToOffsetDateTime {
+
+        @Test
+        fun `should convert Instant to UTC OffsetDateTime`() {
+            // Given
+            val instant = Instant.now()
+            val expected = instant.atOffset(ZoneOffset.UTC)
+
+            // When
+            val actual = instantMapper.toOffsetDateTime(instant)
+
+            // Then
+            assertThat(actual).isEqualTo(expected)
+        }
+    }
```

Comment:
It will be good to also have a test for null instant to null offset date time

Reply:
Added